package.json

```diff
@@ -11,6 +11,8 @@
     "build": "ng build",
     "dartanalyzer": "cd dist/dart && pub install && cd ../.. && ts-node scripts/ci/dart_analyzer",
     "demo-app": "cd src && ng serve",
+    "postinstall": "typings install  --ambient && npm prune",
+    "reinstall": "npm cache clean && rimraf coverage dist typings && rimraf node_modules && npm install",
```

Comment:
We cannot have any `postinstall`/`reinstall` scripts as we still don't have a publishing story. So I would remove `postinstall` and `reinstall` until we know if we can use them. Or @jelbourn maybe it's okay for now since nobody can `npm install material2` (yet).